easybuild/easyconfigs/m/MariaDB/MariaDB-10.0.21-foss-2015b.eb

```diff
@@ -1,27 +1,24 @@
 easyblock = 'CMakeMake'
 
 name = 'MariaDB'
-version = '5.5.29'
```

Comment:
okay, but it's weird. I never moved this file. I don't understand why git thinks it's a rename.